backend/test/users.test.ts

```diff
@@ -645,5 +683,18 @@ describe('user', () => {
         .expect(403);
       expect(response.body).toEqual({});
     });
+    it('update by regular user should not work if changing disabled to false', async () => {
+      const response = await request(app)
+        .put(`/users/${user.id}`)
+        .set(
+          'Authorization',
+          createUserToken({
+            id: user.id
+          })
+        )
+        .send({ firstName, lastName, disabled: false })
+        .expect(403);
+      expect(response.body).toEqual({});
+    });
   });
```

Comment:
@bgtripp I can't seem to find where you implemented this test though. Can you double check to see if you added it? (I believe https://github.com/cisagov/crossfeed/pull/1532/commits/3a8b223b2e325073c28e6d0674a1b7d9ade07596 is different)

Reply:
@epicfaace IIRC, this test in users.test.ts validates that users with `disabled: true` cannot authenticate, and https://github.com/cisagov/crossfeed/commit/3a8b223b2e325073c28e6d0674a1b7d9ade07596 validates that users that have already authenticated but have since been disabled can not get authorization to hit any endpoint.

Reply:
This test in users.test.ts doesn't validate that users with `disabled: true` cannot authenticate. It just validates that users cannot set the `disabled` property.

Reply:
Happy to chat offline if this is unclear!

Reply:
Oops, I meant the test 'verify disabled accounts cannot get tokens' in auth.test.ts validates that users with `disabled: true` cannot authenticate, not this one in users.test.ts.

Reply:
Once the above is done this is ready to go